src/Adapter/Bdf/Shape/TextRenderer.php

```diff
@@ -103,12 +97,10 @@ private static function renderChar(
             }
         }
 
-        $maxX = null;
         foreach ($points as $point) {
-            if (null === $point) {
-                continue;
+            if (null !== $point) {
```

Comment:
this is an "early continue" and is intentional

Reply:
Sure! Sorry about the delay. Done! o/